src/main/java/net/tardis/mod/client/guis/GuiOptifineWarning.java

```diff
@@ -11,32 +11,31 @@
 import net.minecraftforge.fml.common.Mod;
 import net.minecraftforge.fml.common.eventhandler.SubscribeEvent;
 import net.minecraftforge.fml.relauncher.Side;
+import net.tardis.mod.config.TardisConfig;
 
 import java.io.IOException;
 import java.util.List;
 
+import javax.lang.model.util.ElementScanner6;
```

Comment:
Why is this import needed?

Reply:
Didn't add it. I think visual studio code did when it tried auto placing code.